automation/run-tests.sh

```diff
@@ -64,17 +67,31 @@ function install_nmstate {
 }
 
 function run_tests {
-    docker_exec "
-      cd /workspace/nmstate &&
-      pytest \
-        --verbose --verbose \
-        --log-level=DEBUG \
-        --durations=5 \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/libnmstate \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/nmstatectl \
-        --cov-report=html:htmlcov-integ \
-        tests/integration \
-    ${nmstate_pytest_extra_args}"
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "unit_code_style" ];then
+        docker_exec 'cd /workspace/nmstate && tox -e flake8,pylint'
+    fi
+
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "unit_py27" ];then
+        docker_exec 'cd /workspace/nmstate && tox -e check-py27'
+    fi
+
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "unit_py36" ];then
+        docker_exec 'cd /workspace/nmstate && tox -e check-py36'
+    fi
+
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "integ" ];then
+        docker_exec "
```

Comment:
Please use a different function for the integration  and unit tests.

For the unit, you could pass the type as an argument.

Reply:
Why does it matter, whether or not they are integration or unit tests?